src/ssh/azext_ssh/aaz/latest/hybrid_connectivity/endpoint/_list_credential.py

```diff
@@ -0,0 +1,220 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+class ListCredential(AAZCommand):
+    """Gets the endpoint access credentials to the resource.
```

Comment:
Please use the active voice in the help message
```suggestion
    """Get the endpoint access credentials to the resource.
```